src/metax_api/models/catalog_record.py

```diff
@@ -690,6 +700,11 @@ def _pre_create_operations(self):
 
         self.identifier = generate_identifier(urn=False)
 
+        if not self.metadata_owner_org:
```

Comment:
I assume this means that only if metadata_owner_org is empty both in db and in the request, metadata_owner_org gets populated with metadata_provider_org?

If metadata_owner_org exists in db but is not given in the request, maybe it should not change back to metadata_provider_org.

Reply:
I think this would be a legit case for commit squashing, since the later commit is just an improvement for the previous commit.